packages/react-native/src/primitives/RadioGroup/styles.ts

```diff
@@ -1,12 +1,15 @@
 import { StyleSheet } from 'react-native';
 
+import { StrictTheme } from '../../theme';
 import { RadioGroupStyles } from './types';
 
-export const styles: RadioGroupStyles = StyleSheet.create({
-  label: {
-    fontSize: 18,
-    fontWeight: '500',
-    marginHorizontal: 4,
-    marginVertical: 2,
-  },
-});
+export const getThemedStyles = (theme: StrictTheme): RadioGroupStyles => {
+  const { components } = theme.tokens;
+
+  return StyleSheet.create({
+    label: {
+      // The RadioGroup label inherits its styles from the Label primitive
+      ...components?.radiogroup.label,
+    },
```

Comment:
Do we need this? Just thinking it might be confusing for users since `theme` style applied to the `Label` itself would override this? Example:
```
const customTheme = {
  // syntax might be off
  label: { color: 'orange' }, 
  radioGroup: { label: { color: 'red' } },
}
```

That said, think we may still want to provide users the ability to add custom styles for the `container` even if we don't apply any by default

Reply:
Right. Do we want to allow customers to specifically theme the `RadioGroup` label, or should they only be able to theme the `Label` itself?

Reply:
Actually i was off abt my concern. Since these styles are passed via the `style` prop to the `Label` they will take precedence